distributions/docker/solr/src/main/resources/replication-solr-managed-schema.xml

```diff
@@ -124,37 +124,31 @@
 
 
     <!-- replication_item fields -->
-    <field name="resource-modified_tdt" type="pdate" indexed="true" stored="true" required="false" multiValued="false" />
-    <field name="metacard-modified_tdt" type="pdate" indexed="true" stored="true" required="false" multiValued="false" />
-    <field name="source_txt" type="string" indexed="true" stored="true" required="false" multiValued="false" />
-    <field name="destination_txt" type="string" indexed="true" stored="true" required="false" multiValued="false" />
-    <field name="config-id_txt" type="string" indexed="true" stored="true" required="false" multiValued="false" />
-    <field name="failure-count_int" type="pint" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="metadata_id" type="string" indexed="true" stored="true" required="false" multiValued="false" />
```

Comment:
These were all changed from `-` to `_` and removed the `_*` names to better align with solr naming conventions.
https://lucene.apache.org/solr/guide/6_6/defining-fields.html#DefiningFields-FieldProperties